extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -86,24 +97,28 @@ export class AmpAdApiHandler {
     listenForOnce(this.iframe_, 'entity-id', info => {
       this.element_.creativeId = info.id;
     }, this.is3p_);
-    this.unlisteners_.push(listenFor(this.iframe_, 'embed-size', data => {
-      let newHeight, newWidth;
-      if (data.width !== undefined) {
-        newWidth = Math.max(this.element_./*OK*/offsetWidth +
-            data.width - this.iframe_./*OK*/offsetWidth, data.width);
-        this.iframe_.width = newWidth;
-        this.element_.setAttribute('width', newWidth);
-      }
-      if (data.height !== undefined) {
-        newHeight = Math.max(this.element_./*OK*/offsetHeight +
-            data.height - this.iframe_./*OK*/offsetHeight, data.height);
-        this.iframe_.height = newHeight;
-        this.element_.setAttribute('height', newHeight);
-      }
-      if (newHeight !== undefined || newWidth !== undefined) {
-        this.updateSize_(newHeight, newWidth);
-      }
-    }, this.is3p_));
+
+    // Install iframe resize API.
+    this.unlisteners_.push(listenFor(this.iframe_, 'embed-size',
+        (data, source, origin) => {
+          let newHeight, newWidth;
+          if (data.width !== undefined) {
+            newWidth = Math.max(this.element_./*OK*/offsetWidth +
+                data.width - this.iframe_./*OK*/offsetWidth, data.width);
+            this.iframe_.width = newWidth;
+            this.element_.setAttribute('width', newWidth);
+          }
+          if (data.height !== undefined) {
+            newHeight = Math.max(this.element_./*OK*/offsetHeight +
+                data.height - this.iframe_./*OK*/offsetHeight, data.height);
+            this.iframe_.height = newHeight;
+            this.element_.setAttribute('height', newHeight);
+          }
+          if (newHeight !== undefined || newWidth !== undefined) {
+            this.updateSize_(newHeight, newWidth, source, origin);
+          }
+        }, this.is3p_, this.is3p_));
```

Comment:
Yes it is. The option was created with 3P use cases in mind. TBH, I'm not sure amp-ad ever has this set to false, but I'm keeping things consistent.